specification/network/resource-manager/Microsoft.Network/2017-09-01/examples/ApplicationSecurityGroupListReferencingSecurityRules.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters" : {
+    "api-version": "2017-09-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "applicationSecurityGroupName": "test-asg"
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "value": [
```

Comment:
As per the definition of "ResourceReferenceResultList", the items in the value array are of type "ResourceReference" which is an object with one property named {"id": "the reference is mentioned here"}, whereas the example states that the value array is an array of strings. Either the example is correct or the spec. Please rectify the mistake.

Reply:
You're right. The example is wrong. I fixed it accordingly.